style_editor.html

```diff
@@ -1,4 +1,12 @@
+<!DOCTYPE html>
+
+<html lang="en">
```

Comment:
Shouldn't there also be a </html> tag at the end?

Reply:
Yes, someone ate it, fixed.